EnglishTranslation-master/ITEM.tsv

```diff
@@ -10124,7 +10124,7 @@ ITEM_20160224_010123	Contains silver. This item will be removed after the server
 ITEM_20160224_010124	Intermediate Silver Chest
 ITEM_20160224_010125	Advanced Silver Chest
 ITEM_20160224_010126	Settlement Support Potion
-ITEM_20160224_010127	Increases your movement speed by 2, physical and magic attack by 50 for 30 minutes. {nl}Right-click to use.{nl}This item will be removed after the server maintenance of March 4th.
+ITEM_20160224_010127  Increases your movement speed and your physical/magic attack by 2 and 50, respectively, for 30 minutes.{nl}Right-click to use.{nl}This item will be removed after the server maintenance of March 4th.
```

Comment:
Also consider maybe revamping this line a bit more, or reverting it to how it was? what you have is good and the use of "respectively" makes sense, but it might be a bit confusing to a few... so maybe just
`Increases your movement speed by 2 and your physical/magic attack by 50 for 30 minutes`
would suffice here